src/clr-angular/forms/styles/_variables.forms.scss

```diff
@@ -12,9 +12,10 @@ $clr-forms-label-color: $clr-color-neutral-800 !default;
 $clr-forms-text-color: $clr-color-neutral-1000 !default;
 $clr-forms-invalid-color: $clr-color-danger-800 !default;
 $clr-forms-subtext-color: $clr-color-neutral-600 !default;
+$clr-forms-placeholder-color: $clr-color-neutral-600 !default;
 $clr-forms-border-color: $clr-color-neutral-500 !default;
 $clr-forms-focused-color: $clr-color-action-600 !default;
-$clr-forms-disabled-opacity: 0.54 !default;
+$clr-forms-disabled-opacity: 0.55 !default;
```

Comment:
IMO - this shouldn't be hard-coded. 
Could this update the generic forms variable to 0.55 and use that instead?
```--clr-forms-disabled-opacity: #{$clr-forms-disabled-opacity};``` => 0.54. 
It seems where ever $clr-forms-disabled-opacity is needed would also need this enhancement.

Reply:
Sure, will make the needed changes tomorrow and will ping you :)

Reply:
Actually, I want to _remove_ the opacity and shade the label/helper text colors. Right now, this is still inaccessible. https://github.com/vmware/clarity/issues/4378

Reply:
After checking this again, it seems that is already done 

```scss
--clr-forms-disabled-opacity: #{$clr-forms-disabled-opacity}; // 0.55
```

My initial idea was to drop overall the opacity as a way of creating colors and use solid versions of them. But we gonna need more than one color for that. For example, the `border-bottom` of the inputs and the placeholder text color is not the same color. I had one more reason for not doing that when I tested it .. but could not remember right now.

In feature PR could try to add this change but right now the focus of it was the placeholder text inside the input/textarea/selects and to align both the solid color and opacity level to the same value from the design spec.